doc/build/core/temporal.rst

```diff
@@ -0,0 +1,238 @@
+.. _metadata_temporal:
+
+.. currentmodule:: sqlalchemy.schema
+
+==========================================================
+Temporal Constructions (System and Application versioning)
+==========================================================
+
+This section will discuss temporal features added in SQL:2011, which include DDL
+constructs ``PERIOD FOR``, ``PERIOD FOR SYSTEM_TIME``, ``WITH SYSTEM
+VERSIONING``, and ``WITHOUT OVERLAPS``. In SQLAlchemy these constructs can be
+represented using :class:`_schema.Period`, :class:`ApplicationTimePeriod` and
+:class:`.SystemTimePeriod` objects.
+
+
+Context
+-------
+The 2011 release of ISO SQL added support for temporal tables, i.e schemas where
+rows are associated with one or more time periods. This page attemps to give
+only a brief introduction to these new temporal features; for a more complete
+explanation, refer to `TF`_.
+
+.. _TF: https://cs.ulb.ac.be/public/_media/teaching/infoh415/tempfeaturessql2011.pdf
+
+These temporal features are split into system versioning and application
+versioning. System versioning describes row history logging that is handled
+automatically by the backend, useful for producing auditable tables and storing
+history without triggers. Historical rows are (usually) held in the same table,
+and are hidden unless explicitly requested using ``FOR SYSTEM TIME`` in
+``SELECT`` statements.
+
+Application versioning describes roww with a time period indicating its valid
```

Comment:
**Federico Caselli** (CaselIT) wrote:

roww -> row

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/3803